### DISCUSSION TITLE
Broken install of homebrew. Can't uninstall or reinstall now.

### DISCUSSION THREAD

**Discussion by GrueseGehenRaus** [Homebrew/discussions#2069] [Getting started]
Wanted to install cocoa pods via honeybrew but I can't run any brew commands ('zsh: command not found: brew' is what I get in response) I've tried https://github.com/Homebrew/homebrew-core/issues/5084#issuecomment-248834806 to fix it and the initial guide I was following is https://github.com/CocoaPods/CocoaPods/issues/9907#issuecomment-868254538

**Comment by SMillerDev** [comment]
Did you follow the instructions after running the install command for brew?

**Reply by GrueseGehenRaus** [reply]
I couldn't because zsh: command not found: brew was printed into my console

**Reply by SMillerDev** [reply]
Did you try and follow the instructions here: https://brew.sh/? Because those are the official ones.
